packages/components/src/components/TextInput/TextInput.stories.tsx

```diff
@@ -0,0 +1,158 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconIcon, IconPen } from '@frontify/fondue-icons';
+import { action } from '@storybook/addon-actions';
+import { type Meta, type StoryObj } from '@storybook/react';
+import { type ComponentProps } from 'react';
+
+import { Button } from '../Button/Button';
+
+import { TextInput } from './TextInput';
+
+type Story = StoryObj<typeof TextInput>;
+const meta: Meta<typeof TextInput> = {
+    title: 'Components/Text Input',
+    component: TextInput.Root,
+    subcomponents: {
+        // @ts-expect-error Storybook types are incorrect
+        'TextInput.Slot': TextInput.Slot,
+    },
+    tags: ['autodocs'],
+    parameters: {
+        status: {
+            type: 'in_progress',
+        },
+    },
+    args: {
+        placeholder: 'Placeholder',
+        defaultValue: 'Hello from Frontify',
+        onChange: action('onChange'),
+        onBlur: action('onBlur'),
+        onFocus: action('onFocus'),
+    },
+    render: (args) => {
+        // Used to get the correct component in the Storybook for the simple cases (`TextInput` instead of `TextInput.Root`)
```

Comment:
this is only used once, why not define this in the story and use one of the more commonly used ones as the component default?

Reply:
Prefer to have the "simple" and most used case as the default `render`